drivers/timer/stm32_lptim_timer.c

```diff
@@ -174,6 +180,8 @@ void sys_clock_set_timeout(int32_t ticks, bool idle)
 	else if (next_arr < (lp_time + LPTIM_GUARD_VALUE)) {
 		next_arr = lp_time + LPTIM_GUARD_VALUE;
 	}
+	/* with slow lptim_clock, LPTIM_GUARD_VALUE of 1 is enough */
+	next_arr = next_arr - 1;
```

Comment:
Houls it be moved to `drivers: timer: stm32 adjust lptimer with slow LPTIM clock` commit?

Reply:
yes, done